tests/rule_based_profiler/parameter_builder/test_mean_unexpected_map_metric_multi_batch_parameter_builder.py

```diff
@@ -188,13 +200,121 @@ def test_mean_unexpected_map_metric_multi_batch_parameter_builder_bobby_datetime
+        reduce_scalar_metric=True,
+        evaluation_parameter_builder_configs=None,
+        json_serialize=False,
+        batch_list=None,
+        batch_request=batch_request,
+        data_context=data_context,
+    )
+
+    evaluation_parameter_builder_configs: Optional[List[ParameterBuilderConfig]] = [
+        ParameterBuilderConfig(
+            **my_total_count_metric_multi_batch_parameter_builder.to_json_dict()
+        ),
+        ParameterBuilderConfig(
+            **my_null_count_metric_multi_batch_parameter_builder.to_json_dict()
+        ),
+    ]
+    mean_unexpected_map_metric_multi_batch_parameter_builder: MeanUnexpectedMapMetricMultiBatchParameterBuilder = MeanUnexpectedMapMetricMultiBatchParameterBuilder(
+        name="my_passenger_count_values_not_null_mean_unexpected_map_metric",
+        map_metric_name="column_values.nonnull",
+        total_count_parameter_builder_name="my_total_count",
+        null_count_parameter_builder_name="my_null_count",
+        metric_domain_kwargs=DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
+        metric_value_kwargs=None,
+        evaluation_parameter_builder_configs=evaluation_parameter_builder_configs,
+        json_serialize=False,
+        batch_list=None,
+        batch_request=batch_request,
+        data_context=data_context,
+    )
+
+    metric_domain_kwargs: dict = {"column": "passenger_count"}
+    domain: Domain = Domain(
+        domain_type=MetricDomainTypes.COLUMN, domain_kwargs=metric_domain_kwargs
+    )
+
+    parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)
+
+    variables: Optional[ParameterContainer] = None
+    parameters: Dict[str, ParameterContainer] = {
+        domain.id: parameter_container,
+    }
+
+    mean_unexpected_map_metric_multi_batch_parameter_builder.build_parameters(
+        parameter_container=parameter_container,
+        domain=domain,
+        variables=variables,
+        parameters=parameters,
+    )
+
+    expected_parameter_value: float = 0.0
+
+    actual_parameter_value: Optional[
+        Any
+    ] = get_parameter_value_and_validate_return_type(
+        domain=domain,
+        parameter_reference=mean_unexpected_map_metric_multi_batch_parameter_builder.fully_qualified_parameter_name,
+        expected_return_type=None,
+        variables=variables,
+        parameters=parameters,
+    )
+
+    rtol: float = RTOL
+    atol: float = 5.0e-1 * ATOL
+    np.testing.assert_allclose(
+        actual=actual_parameter_value.value,
+        desired=expected_parameter_value,
+        rtol=rtol,
+        atol=atol,
+        err_msg=f"Actual value of {actual_parameter_value.value} differs from expected value of {expected_parameter_value} by more than {atol + rtol * abs(actual_parameter_value.value)} tolerance.",
+    )
+
+
+def test_mean_unexpected_map_metric_multi_batch_parameter_builder_bobby_datetime_dependencies_evaluated_separately(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
```

Comment:
Same note here and throughout. Any usage of Alice/Bobby/Bobster/Quentin should be integration.

Reply:
@cdkini We can do this as a separate PR (along with other tests that are similarly inefficient).  Thanks.  P.S.: Could be a nice hackathon project! :-)